### PR TITLE
Add reslice cursor widget

### PR DESCRIPTION
**Pull request by luciemac**

### CODE REVIEW

**Comment by luciemac** [comment]
@agirault can you merge please?

**Comment by jourdain** [comment]
Travis is failing...

File: Sources/Filters/Sources/PlaneSource/index.js [hunk 1]
```diff
@@ -115,12 +261,20 @@ const DEFAULT_VALUES = {
...
   macro.setGet(publicAPI, model, ['xResolution', 'yResolution']);
-  macro.setGetArray(publicAPI, model, ['origin', 'point1', 'point2'], 3);
+  macro.setGetArray(publicAPI, model, ['origin'], 3);
+  macro.getArray(publicAPI, model, ['point1', 'point2', 'normal', 'center'], 3);
+
   macro.algo(publicAPI, model, 0, 1);
   vtkPlaneSource(publicAPI, model);
+
+  publicAPI.setPoint1(model.point1);
+  publicAPI.setPoint1(model.point2);
```
Comment (jmargeta) [inline Sources/Filters/Sources/PlaneSource/index.js]: The second point is not being set here (just a copy-paste typo), should be publicAPI.setPoint**2**(model.point2);

**Comment by finetjul** [comment]
Travis is failing

**Comment by luciemac** [comment]
@finetjul can you review, please ?

**Comment by finetjul** [comment]
Looks good to me 👍

**Pull request merged**